vrs/vrs_annotation_batch.py

```diff
@@ -71,7 +71,8 @@ def main(args):
 
     # Initialize Hail w/ Query-On-Batch , setting global seed if user decides to downsample
     hl.init(
```

Comment:
should raise a ValueError if --run-vrs is set and and backend_mode is set to spark (as --run-vrs should require using batch)